.github/workflows/cgl.yml

```
@@ -0,0 +1,61 @@
+name: cgl.yml
+on:
```

actionlint (1.7.7)

string should not be empty

(syntax-check)